File: 240421/코로나 메뉴얼/covid-manual.js
class Patient {
    #hasSymptom;
    #temperature;

    constructor(hasSymptom, temperature){
        this.#hasSymptom = hasSymptom;
        this.#temperature = temperature;
    }

    goAClinic(){
        return this.#hasSymptom && this.#temperature >= 37;
    }

}

function emergencyJudgment() {
    const fs = require('fs');

    const arr = fs.readFileSync(0).toString().trim().split(/\s+/);
    
    const p1 = new Patient(hasSymptom(arr[0]), arr[1]);
    const p2 = new Patient(hasSymptom(arr[2]), arr[3]);
    const p3 = new Patient(hasSymptom(arr[4]), arr[5]);
    
    let cnt = 0;

    if(p1.goAClinic()){
        cnt++;
    }

    if(p2.goAClinic()){
        cnt++;
    }

    if(p3.goAClinic()){
        cnt++;
    }

    if(cnt >= 2){
        console.log('E');
    }else{
        console.log('N');
    }

}

function hasSymptom(value){
    return value==='Y' ? true : false;
}

emergencyJudgment();

File: 240421/코로나 메뉴얼/covid-manual.java
import java.util.*;

public class Main {
    public static class Patient {
        boolean hasSymptom;
        int temperature;

        public Patient(boolean hasSymptom, int temperature){
            this.hasSymptom = hasSymptom;
            this.temperature = temperature;
        }

        public boolean goAClinic(){
            return hasSymptom && temperature >= 37;
        }

    }
    public static void main(String[] args) {
        Scanner sc = new Scanner(System.in);
        
        Patient p1 = new Patient(hasSymptom(sc.next()), sc.nextInt());
        Patient p2 = new Patient(hasSymptom(sc.next()), sc.nextInt());
        Patient p3 = new Patient(hasSymptom(sc.next()), sc.nextInt());
        
        int cnt = 0;

        if(p1.goAClinic()){
            cnt++;
        }

        if(p2.goAClinic()){
            cnt++;
        }

        if(p3.goAClinic()){
            cnt++;
        }

        if(cnt >= 2){
            System.out.println('E');
        }else{
            System.out.println('N');
        }

    }

    public static boolean hasSymptom(String value){
        return value.equals("Y") ? true : false;
    }
}